src/components/common/Banner.jsx

```
@@ -59,17 +60,20 @@ const Banner = ({ className = '' }) => {
           <img
             src={isMobile ? MobileCoffeeBanner : WebCoffeeBanner}
             alt='커피 이벤트 배너'
+            fetchpriority='high'
```

ESLint

[error] Unknown property 'fetchpriority' found, use 'fetchPriority' instead

(react/no-unknown-property)